doc/source/vmeasure.rst

```diff
@@ -0,0 +1,36 @@
+V-measure
+=============
+
+The V-Measure is defined as the harmonic mean of homogeneity :math:`h` and completeness :math:`c` of the clustering. Both these measures can be expressed in terms of the mutual information and entropy measures originating from the field of information retrieval.
```

Comment:
IMHO "originating from the field of information retrieval" -> "of the information theory" or something like that